### PR TITLE
Mod 9197 word of the day error

### PR DESCRIPTION
**Pull request by chasls2**
**High level description:**

- Changed the Error Message for the WordOfTheDay component. 
- Added some wording and a button to send an email for support
- created another css class for WordOfTheDay component to change the styling if its an error message

**Technical details:**

**JIRA Ticket:**
[DEV-9197](https://federal-spending-transparency.atlassian.net/browse/DEV-9197)

**Mockup:**
https://invis.io/RYA3XN5WP#/273832670_Homepage_2-2_E

The following are ALL required for the PR to be merged:

Author:
- [x] Linked to this PR in JIRA ticket
- [ ] Scheduled demo including Design/Testing/Front-end OR Provided instructions for testing in JIRA and PR `if applicable`
- [x] Verified cross-browser compatibility: Chrome, Safari, Firefox, Edge
- [x] Verified mobile/tablet/desktop/monitor responsiveness
- [x] Verified that this PR does not create any *new* accessibility issues (via Axe Chrome extension)
- [ ] Added Unit Tests for helper functions, reducers, models and Container/Component Interactivity Expectations `if applicable` [React Testing Library](react-testing-library.md)
- [ ] [API contract](https://github.com/fedspendingtransparency/usaspending-api/tree/dev/usaspending_api/api_contracts) updated `if applicable`
- [ ] [Component Library Integration Status Report](https://github.com/fedspendingtransparency/data-act-documentation/blob/data-transparency-ui/frontend_apps/component-library-integration-status.md) updated `if applicable`

Reviewer(s):
- [ ] Design review complete `if applicable`
- [ ] [API #1234](https://github.com/fedspendingtransparency/usaspending-api/pull/1234) merged concurrently `if applicable`
- [ ] Code review complete

### CODE REVIEW

File: src/js/components/homepageUpdate/WordOfTheDay/ErrorWordOfTheDay.jsx [hunk 1]
```diff
@@ -7,13 +7,10 @@ import React from 'react';
...
-            <img
-                role="presentation"
-                src="../../../../img/homepage-word-of-the-day/Error-Placeholder.svg"
-                alt="" />
-        </picture>
-        Unable to load Word of the Day
+        <p className="word-of-the-day-paragraph-one">Something went wrong</p>
+        <p className="word-of-the-day-paragraph-two">Sorry, we're unable to load this content</p>
+        {/* eslint-disable-next-line no-return-assign */}
+        <button className="word-of-the-day-button" onClick={() => window.location = 'mailto:usaspending.help@fiscal.treasury.gov?subject=Word%20of%20the%20Day$20Error'}>Report this error</button>
```
Comment (anjenkin) [inline src/js/components/homepageUpdate/WordOfTheDay/ErrorWordOfTheDay.jsx:13]: There's a typo in the last %20 so the subject shows up incorrectly

Screen Shot 2022-10-27 at 8 11 19 AM

File: src/_scss/pages/homepageUpdate/_wordOfTheDay.scss [hunk 2]
```diff
@@ -104,5 +107,24 @@
...
+    font-weight: $font-semibold;
+    margin-bottom: rem(4);
+  }
+  .word-of-the-day-paragraph-two{
+    font-size: rem(16);
+    margin-top: rem(4);
+  }
+  .word-of-the-day-button{
+    border-radius: rem(4);
+    border: solid rem(1) #dfe1e2;
```
Comment (anjenkin) [inline src/_scss/pages/homepageUpdate/_wordOfTheDay.scss:124]: color variables from _scss/cores/variables.scss could be used here for the border and background-color

File: src/_scss/pages/homepageUpdate/_wordOfTheDay.scss [hunk 3]
```diff
@@ -104,5 +107,24 @@
 
   .word-of-the-day__error-message {
     text-align: center;
+    color: $gray-2;
+  }
+  .word-of-the-day-paragraph-one{
+    font-size: rem(24);
```
Comment (anjenkin) [inline src/_scss/pages/homepageUpdate/_wordOfTheDay.scss:113]: font size variables from _scss/core/variables.scss can be used here and in the word-of-the-day-paragraph-two class below

**Review by anjenkin — CHANGES_REQUESTED** [review]
looks good.  just a few comments inline

File: src/js/components/homepageUpdate/WordOfTheDay/ErrorWordOfTheDay.jsx (hunk 1 above)
Comment (chasls2) [reply src/js/components/homepageUpdate/WordOfTheDay/ErrorWordOfTheDay.jsx]: fixed

File: src/_scss/pages/homepageUpdate/_wordOfTheDay.scss (hunk 2 above)
Comment (chasls2) [reply src/_scss/pages/homepageUpdate/_wordOfTheDay.scss]: Changed to color  var

File: src/_scss/pages/homepageUpdate/_wordOfTheDay.scss (hunk 3 above)
Comment (chasls2) [reply src/_scss/pages/homepageUpdate/_wordOfTheDay.scss]: changed to font size vars. I changed the other paragraph as well.

**Comment by anjenkin** [comment]
@chasls2 Looks good for the most part now.  One last thing... I just noticed that the text can be centered on tablet and mobile.  
Screen Shot 2022-10-27 at 9 47 00 AM

**Review by anjenkin — APPROVED** [review]

File: src/js/components/homepageUpdate/WordOfTheDay/WordOfTheDay.jsx [hunk 4]
```diff
@@ -172,7 +172,7 @@ const WordOfTheDay = () => {
                 selectWordOfTheDay();
                 setGlossary(res.data.results);
                 setLoading(false);
-                setError(false);
+                setError(true);
```
Comment (anjenkin) [inline src/js/components/homepageUpdate/WordOfTheDay/WordOfTheDay.jsx:175]: Looks like this was missed.  The word of the date error message is always showing now.

**Review by anjenkin — CHANGES_REQUESTED** [review]
See additional comment inline.  The Word of the Day card is always showing an error now.

File: src/js/components/homepageUpdate/WordOfTheDay/WordOfTheDay.jsx (hunk 4 above)
Comment (chasls2) [reply src/js/components/homepageUpdate/WordOfTheDay/WordOfTheDay.jsx:175]: on it

**Review by anjenkin — APPROVED** [review]

**Pull request merged**